mindsdb/integrations/handlers/paypal_handler/paypal_tables.py

```diff
@@ -137,21 +138,21 @@ def select(self, query: ast.Select) -> pd.DataFrame:
         )
         selected_columns, where_conditions, order_by_conditions, result_limit = select_statement_parser.parse_query()
 
-        search_params = {"ids": []}
+        id=""
         subset_where_conditions = []
         for op, arg1, arg2 in where_conditions:
-            if arg1 == 'ids':
+            if arg1 == 'id':
                 if op == '=':
-                    search_params["ids"] = arg2
+                    id=arg2
                 else:
                     raise NotImplementedError("Only '=' operator is supported for 'ids' column")
             elif arg1 in ['state', 'amount', 'create_time', 'update_time', 'links', 'pending_reason', 'parent_payment']:
```

Comment:
Rather than doing this, let's remove "id" from `where_conditions` once the `for` loop has been completed.

Reply:
u mean:
        add `where_conditions=[]` after for loop?

Reply:
Actually, let's forget this. What you have done here is correct.